Makefile

```diff
@@ -42,11 +58,25 @@ user-authenticate:
 	@read -p "User name: " uname; \
 	$(PYTHON) $(PYDIR)/manage.py drf_create_token $$uname --settings=config.settings.local
 
+# There is a race condition with how docker applies MCS
+# categories on SELinux-enabled systems. So, we need to ensure
+# the app server starts up after the web proxy.
+#
+# Users should also set:
+# setsebool -P container_connect_any on
 start-compose:
-	docker-compose up --build -d
+	@if [ $(FLAVOR) == 'redhat' ]; then \
+		docker-compose up --build -d web db queue; \
+		$(SUDO) chmod 777 $(SOCKETDIR); \
+		$(SUDO) chcon -l s0 $(SOCKETDIR); \
+		docker-compose up --build -d app; \
+	else \
+		docker-compose up --build -d; \
+	fi
 
 stop-compose:
 	docker-compose down
+	$(SUDO) rm -rf $(TMPDIR)
```

Comment:
I don't think destroying directories makes sense in `stop-compose`. If anywhere, I think it should go in `clean`, but even then I don't like the idea of a make command destroying *all* of my `TMPDIR` since other processes might be putting stuff there. Maybe target the specific files you want gone instead.